python_modules/dagster/dagster_tests/core_tests/instance_tests/test_instance.py

```diff
@@ -569,6 +572,44 @@ def test_dagster_home_not_dir():
             DagsterInstance.get()
 
 
+def test_dagster_env_vars_from_dotenv_file():
```

Comment:
this test is an example of both kinds of Dagster env vars: DAGSTER_HOME itself, plus SQLITE_STORAGE_BASE_DIR , referenced in the dagster.yaml file